recipes-core/tdx-enc-handler/tdx-enc-handler/tdx-enc-caam.sh

```diff
@@ -14,8 +14,8 @@ TDX_ENC_STORAGE_MOUNTPOINT="@@TDX_ENC_STORAGE_MOUNTPOINT@@"
 # dm-crypt device to be created
 TDX_ENC_DM_DEVICE="encdata"
 
-# CAAM black key name
-TDX_ENC_KEY_NAME="dek"
+# CAAM encrypted key name
+TDX_ENC_KEY_NAME="dek.bb"
```

Comment:
Is `.bb` a common extension for this usage here?

Reply:
Hmmm, I am not sure if there is a common extension for this use case. I am just following here what NXP has used in its manual.

`bb` stands for black blob, and that is how they call the blob that has the encrypted key.

Anyway, naming stuff is hard, so I am open to new ideas. :-)

Reply:
Okay. It just felt kind of funny to see the same extension used with BitBake recipes :slightly_smiling_face: